.changes/next-release/feature-AWSSDKforJavav2-787c575.json

```diff
@@ -0,0 +1,6 @@
+{
+    "type": "feature",
+    "category": "AWS SDK for Java v2",
+    "contributor": "",
+    "description": "Make Transfer Manager work by default with MultipartS3AsyncClient."
```

Comment:
`MultipartS3AsyncClient` is an internal API. Can we say something like `S3AsyncClient with multipart configuration enabled`?